Inputs/Input_Info.py

```diff
@@ -7,7 +7,7 @@
 
 ################################
 # Name of output directories
-datacardInputs = "datacardInputs"
+datacardInputs = "{year}/datacardInputs"
```

Comment:
I think this way should be better. So, that whatever the name of directory it will be going to propagate automatically. 

Else we need to add {year} manually inside the code everywhere.